src/service.ts

```diff
@@ -25,4 +26,10 @@ export default async function init({
       };
     }
   });
+
+  iris.register({
+    pattern: 'action.annotate.metis',
+    handler: annotate
+  });
```

Comment:
@Istar-Eldritch I don't quite understand how this is supposed to be done. This gives me an error.

Reply:
I'll check the PR code locally, to understand what you talk about.